src/display/api.js

```diff
@@ -1046,6 +1047,9 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
       }, this);
       this.objs.clear();
       this.annotationsPromise = null;
+      if (resetStats && this._stats instanceof StatTimer) {
+        this._stats = new StatTimer();
```

Comment:
Wouldn't it be better to implement a `cleanup` method on the `StatTimer`  so we can avoid creating a new instance here and re-use the existing one?

Reply:
While I'm not sure if creating a new instance is that much of an issue in practice, since it will only occur during testing, it's nonetheless a really nice idea!

I've updated the patch with `reset` methods in `StatTimer`/`DummyStatTimer` (in the first case, it's also being re-used by the `constructor`).

Reply:
The `reset` method is good and the re-use is also nice.